src/index.ts

```diff
@@ -700,7 +746,7 @@ async function distributeFnTransactional(
           if (Object.keys(finalDoc).length > 0 && !skipRunViewLogics &&
               ["create", "merge"].includes(action)) {
             logicDoc.doc = finalDoc;
-            await queueRunViewLogics(logicDoc);
+            logicDocsForQueuing.push(logicDoc);
```

Comment:
No need to collect logicDocsForQueuing since they are all part of the logicResults